.github/workflows/safe-transaction.yml

```
@@ -0,0 +1,87 @@
+name: 'Safe Transaction Proposer'
+on:
+  workflow_call:
+    inputs:
+      rpc-url:
+        description: 'RPC URL for the blockchain network'
+        required: true
+        type: string
+      chain-id:
+        description: 'Chain ID of the blockchain network (default: 42161 for Arbitrum)'
```

It's safer to have everything set explicitly
```suggestion
        description: 'Chain ID of the blockchain network'
```

---

Chain ID param was removed and automatically detected from the rlc_URL
Fix Commit: ee4652ba92299916fb022f681d474df1db6fa2da